common/include/pcl/common/centroid.h

```diff
@@ -589,6 +589,68 @@ namespace pcl
     return (computeCovarianceMatrix<PointT, double> (cloud, indices, covariance_matrix));
   }
 
+
+  /** \brief Compute centroid, OBB, PCA axes of a given set of points.
+  * OBB is oriented like the three axes (major, middle and minor) with
+  * major_axis  = obb_rotational_matrix.col(0)
+  * middle_axis = obb_rotational_matrix.col(1)
+  * minor_axis  = obb_rotational_matrix.col(2)
+  * one way to visualize OBB when Scalar is float:
+  * Eigen::Vector3f position(obb_position(0), obb_position(1), obb_position(2));
+  * Eigen::Quaternionf quat(obb_rotational_matrix);
+  * viewer->addCube(position, quat, obb_dimensions(0), obb_dimensions(1), obb_dimensions(2), .....);
+  * \param[in] cloud the input point cloud
+  * \param[out] centroid the centroid of the set of points in the cloud
+  * \param[out] obb_position position of the centroid of the OBB (Oriented Bounding Box)
+  * \param[out] obb_dimensions (width, height and depth) of the OBB (Oriented Bounding Box)
+  * \param[out] obb_rotational_matrix rotational matrix of the OBB (Oriented Bounding Box)
+  * \return number of valid points used to determine the output.
+  * In case of dense point clouds, this is the same as the size of the input cloud.
+  * \ingroup common
+  */
+
+
+  template <typename PointT, typename Scalar> inline unsigned int
+    computeCentroidAndOBB(const pcl::PointCloud<PointT>& cloud,
+                    Eigen::Matrix<Scalar, 3, 1>& centroid,
+                    Eigen::Matrix<Scalar, 3, 1>& obb_position,
+                    Eigen::Matrix<Scalar, 3, 1>& obb_dimensions,
+                    Eigen::Matrix<Scalar, 3, 3>& obb_rotational_matrix);
+
+
+ 
+
```

Comment:
Please limit blank lines to one or two, and no blank lines between a function and its documentation

Reply:
ok